src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/TestRequestHandler.cs

```diff
@@ -84,6 +84,10 @@ public void ProcessRequests(ITestHostManagerFactory testHostManagerFactory)
             do
             {
                 var message = this.communicationManager.ReceiveMessage();
+                if (EqtTrace.IsInfoEnabled)
```

Comment:
You mean verbose? Don't think printing JSON protocol messages are informational :)

Reply:
This data is limited and important. Keep it as Info.